examples/storybook/.storybook/typings.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.md' {
```

Comment:
which markdown is imported?

Reply:
that came from the [Angular Template](https://github.com/storybookjs/storybook/blob/next/lib/cli/src/generators/ANGULAR/template-csf/.storybook/typings.d.ts) of storybook, and it looks like they include it in many templates. 
We don't have any markdown stories in this example.

Reply:
I guess it's related to compodoc, but we removed that now, so this can be gone too?

Reply:
I tried removing compodoc but it doesn't look like it's supposed to work without it. I can fake the documentation.json file but I think it's a better choice the keep the Angular Template defaults no?

https://app.circleci.com/pipelines/github/algolia/angular-instantsearch/252/workflows/6a085af4-3441-4409-b3ed-aae957196913/jobs/8779

Reply:
all right, we can leave the defaults if it doesn't work without, investigate another time otherwise :)

Reply:
I finally managed to remove it 😓  you're right it's cleaner. 2e494d6